### DISCUSSION TITLE
com.apple.iCloudHelper wants to use the "fastlane_tmp_keychain-db" keychain

### DISCUSSION THREAD

**Discussion by alexcroox** [fastlane/fastlane#21259] [Q&A] [answered]
Every hour or so since setting up my local macOS dev machine to be a run target for GitHub actions I get this popup:

![image](https://github.com/fastlane/fastlane/assets/1065835/c7063557-1269-4cc6-b25c-c02c702882f1)

The correct password is blank, but I get it several times a day at seemingly random times, any idea how to stop it?

![image](https://github.com/fastlane/fastlane/assets/1065835/cab78939-75dc-46f3-bb9f-61aed50510fd)

I can untick lock every x minutes but I think Fastlane destroys and re-creates this keychain on every build so I'm back to square one every few days 

![image](https://github.com/fastlane/fastlane/assets/1065835/76f34699-dce6-4f27-9fe4-277356a7a841)

**Comment by GoMino** [comment]
same issue here

**Comment by alexcroox** [comment, marked as answer]
OK found an answer. Select the fastlane keychain, go to the menubar menu and click edit > delete.

Be sure to copy/paste any items in there to your login/icloud one as the fastlane one was a default keychain for months for me so it had other non fastlane related items in there 

https://stackoverflow.com/questions/68461903/how-to-un-set-fastlane-tmp-keychain-db-as-the-default-keychain
